src/features/defi/contexts/FoxyProvider/FoxyProvider.tsx

```diff
@@ -1,41 +0,0 @@
-import type { FoxyApi } from '@shapeshiftoss/investor-foxy'
-import type { PropsWithChildren } from 'react'
-import React, { useContext, useEffect, useState } from 'react'
-import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
-import { logger } from 'lib/logger'
-import { getFoxyApi } from 'state/apis/foxy/foxyApiSingleton'
-
-const moduleLogger = logger.child({ namespace: ['FoxyProvider'] })
-
-type FoxyContextProps = {
-  loading: boolean
```

Comment:
If we want to properly implement this, can do so by selecting pending queries either with provider granularity or for all providers depending on what we're trying to achieve